include/deal.II/base/parameter_handler.h

```diff
@@ -67,10 +67,19 @@ namespace Patterns
      */
     virtual bool match (const std::string &test_string) const = 0;
 
+    enum OutputStyle
+    {
+      // Machine interpretable format, default and used for create
+      Machine,
```

Comment:
Can you elaborate when "used for create" is supposed to mean? Also,
1/ We typically use lower-case names for enums
2/ The comments in front of the enums are copied into doxygen output if you use `/** ... */` style